### PR TITLE
controller insert method homework: tests, dto utils, postman collection

### PR DESCRIPTION
**Pull request by u2107680007911543774e7r**

### CODE REVIEW

File: src/main/java/dto/CustomerDetails.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package dto;
+
+import java.math.BigDecimal;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+public class CustomerDetails {
+	@Valid
```
Comment (Daniluko) [inline src/main/java/dto/CustomerDetails.java]: I think it's don't need

File: src/main/java/dto/CustomerDetails.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	@NotNull(message = "3")
+	private String company;
+	@Valid
```
Comment (Daniluko) [inline src/main/java/dto/CustomerDetails.java]: Also

File: src/main/java/dto/CustomerRequest.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+import java.math.BigDecimal;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.Max;
+
+public class CustomerRequest {
+	@Valid
```
Comment (Daniluko) [inline src/main/java/dto/CustomerRequest.java]: Also

File: src/test/java/service/CustomerServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.internal.matchers.Null;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import entity.Customer;
+import repository.CustomerRepository;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CustomerServiceTest {
```
Comment (Daniluko) [inline src/test/java/service/CustomerServiceTest.java:33]: why do you add this class?

**Review by Daniluko — APPROVED** [review]

File: src/main/java/dto/CustomerDetails.java (hunk 2 above)
Comment (SerhiiPasko) [reply src/main/java/dto/CustomerDetails.java]: Yes and you twice wrote "@NotNull(message = "3")"

File: src/test/java/util/CustomerDtoModelsUtilExist.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
+package util;
+
+import java.math.BigDecimal;
+
+import dto.CustomerDetails;
+import dto.CustomerRequest;
+import entity.Customer;
+
+public class CustomerDtoModelsUtilExist {
```
Comment (SerhiiPasko) [inline src/test/java/util/CustomerDtoModelsUtilExist.java]: It is bad way, test before was ok

Comment (SerhiiPasko) [reply src/test/java/util/CustomerDtoModelsUtilExist.java]: We create util for validation test only becouse must meni times use it in tests

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Read comments, also Rostik's too

**Review by SerhiiPasko — COMMENTED** [review]
Good

**Pull request merged**